### PR TITLE
Merge latest commits from Sufia

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

**Comment by jcoyne** [comment]
Here's an interesting error:
```
 class PermissionTemplateChangeColumnWorkflowName < ActiveRecord::Migration[4.2] (called from block in execute at /home/travis/.rvm/gems/ruby-2.4.0@global/gems/rake-12.0.0/lib/rake/task.rb:250)

== 20170320192792 PermissionTemplateChangeColumnWorkflowName: migrating =======

-- change_column_null(:permission_templates, :workflow_name, false)

rake aborted!

StandardError: An error has occurred, this and all later migrations canceled:

undefined method `null=' for nil:NilClass

/home/travis/.rvm/gems/ruby-2.4.0/gems/activerecord-5.0.2/lib/active_record/connection_adapters/sqlite3_adapter.rb:400:in `block in change_column_null'

/home/travis/.rvm/gems/ruby-2.4.0/gems/activerecord-5.0.2/lib/active_record/connection_adapters/sqlite3_adapter.rb:435:in `block in alter_table'

/home/travis/.rvm/gems/ruby-2.4.0/gems/activerecord-5.0.2/lib/active_record/connection_adapters/sqlite3_adapter.rb:467:in `block in copy_table'

/home/travis/.rvm/gems/ruby-2.4.0/gems/activerecord-5.0.2/lib/active_record/connection_adapters/abstract/schema_statements.rb:272:in `create_table'

/home/travis/.rvm/gems/ruby-2.4.0/gems/activerecord-5.0.2/lib/active_record/connection_adapters/sqlite3_adapter.rb:452:in `copy_table'

/home/travis/.rvm/gems/ruby-2.4.0/gems/activerecord-5.0.2/lib/active_record/connection_adapters/sqlite3_adapter.rb:445:in `move_table'

/home/travis/.rvm/gems/ruby-2.4.0/gems/activerecord-5.0.2/lib/active_record/connection_adapters/sqlite3_adapter.rb:440:in `block in alter_table'

/home/travis/.rvm/gems/ruby-2.4.0/gems/activerecord-5.0.2/lib/active_record/connection_adapters/abstract/database_statements.rb:230:in `transaction'

/home/travis/.rvm/gems/ruby-2.4.0/gems/activerecord-5.0.2/lib/active_record/connection_adapters/sqlite3_adapter.rb:437:in `alter_table'

/home/travis/.rvm/gems/ruby-2.4.0/gems/activerecord-5.0.2/lib/active_record/connection_adapters/sqlite3_adapter.rb:399:in `change_column_null'
```

**Comment by jcoyne** [comment]
@mjgiarlo this might be because we use Sipity::Workflow#permission_template_id. in Hyrax. You can just dump that migration.

File: db/migrate/20170317141521_permission_template_change_column_workflow_name.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class PermissionTemplateChangeColumnWorkflowName < ActiveRecord::Migration
```
Comment (jcoyne) [inline db/migrate/20170317141521_permission_template_change_column_workflow_name.rb]: We don't want this migration in Hyrax.

Comment (mjgiarlo) [inline db/migrate/20170317141521_permission_template_change_column_workflow_name.rb]: Duh, right.

**Comment by jcoyne** [comment]
@mjgiarlo It looks like the jasmine tests are failing

**Comment by mjgiarlo** [comment]
@jcoyne oh, shoot, I deleted the TestResponses file. I'll restore it and push again.

**Comment by mjgiarlo** [comment]
@jcoyne :clap:

**Pull request merged**